diff -Naur mysqlcppapi-1.9.3/mysqlcppapi/Connection.cc mysqlcppapi-1.9.3-p/mysqlcppapi/Connection.cc
--- mysqlcppapi-1.9.3/mysqlcppapi/Connection.cc	2003-08-21 21:13:46.000000000 +0200
+++ mysqlcppapi-1.9.3-p/mysqlcppapi/Connection.cc	2006-01-11 12:17:24.837438444 +0100
@@ -212,7 +212,7 @@
 {
   check_connection_is_open();
   
-  bool suc = !(mysql_shutdown(m_sharedptr_connection.obj()));
+  bool suc = !(mysql_shutdown(m_sharedptr_connection.obj(), SHUTDOWN_DEFAULT));
 
   if (!suc)
     throw ex_BadQuery(error());
